### DISCUSSION TITLE
dotnet 8 for npm

### DISCUSSION THREAD

**Discussion by KarolZirex** [RicoSuter/NSwag#4616] [General]
Do you know maybe when would be 14.0.0-preview010 with support dotnet8 for npm ready?
For now I have  npm version 14.0.0-preview003 and nswag doesn't work for me :/

> Error converting value "Net80" to type 'NSwag.Commands.Runtime'. Path 'runtime', line 2, position 20.

**Comment by lahma** [comment]
This will probably fix the issue when merged https://github.com/RicoSuter/NSwag/pull/4614
